buildpack_integration_test.go

```diff
@@ -0,0 +1,162 @@
+package acceptance_test
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/paketo-buildpacks/occam"
+	. "github.com/paketo-buildpacks/occam/matchers"
+	"github.com/paketo-buildpacks/packit/v2/pexec"
+)
+
+const lifecycleVersion = "0.14.0"
```

Comment:
I see you already have a TODO for this that's checked off, but let's stop using hardcoded lifecycle